src/frontend/utils/validators.ts

```diff
@@ -45,3 +45,8 @@ export const erGyldigDato = (felt: IFelt<IBarnBeregning>): IFelt<IBarnBeregning>
         ? ok(felt)
         : feil(felt, 'Ugyldig dato');
 };
+
+export const erGyldigBegrunnelse = (felt: IFelt<string>): IFelt<string> => {
+    //TODO: decent validation
```

Comment:
Just check if the field is not '' at least to begin with. We can set the rules later.

Reply:
ok